### PR TITLE
Fix --quiet

### LINKED ISSUE #815: cli: singularity build should not print logs to terminal

### PR DESCRIPTION
**Pull request by ishanashah**
Fixes #815. @ivotron your feedback would be appreciated

### CODE REVIEW

**Review by ivotron — CHANGES_REQUESTED** [review]
thanks @ishanashah! The logging level for the lines you modified was already OK. The main problem for the bug you're trying to address is in [this line](https://github.com/getpopper/popper/blob/master/src/popper/runner_host.py#L488) . What needs to be done for this issue is to find a way for that line to _not_ produce output when `--quiet` is given.

**Comment by ishanashah** [comment]
I set the "quiet" parameter in self._s.build to pass information to the build method about whether low priority logs should commit. The Singularity source code allows this parameter to be set, but it seems to not be working on the Singularity side. This fix should solve the issue on the side of Popper, but another fix in the Singularity source code is required.

File: src/popper/runner_host.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def run(self, step):
         if not container and not self._config.reuse:
             container = self._create_container(cid, step)
 
-        log.info(f"[{step.id}] docker start")
+        log.step_info(f"[{step.id}] docker start")
```
Comment (ivotron) [inline src/popper/runner_host.py:150]: these were OK as `log.info()`, this are supposed to still be shown when `--quiet` is given

File: src/popper/runner_host.py [hunk 2]
```diff
@@ -485,7 +485,13 @@ def _build_from_recipe(self, build_ctx_path, build_dest, cid):
         pwd = os.getcwd()
         os.chdir(build_ctx_path)
         recipefile = SingularityRunner._get_recipe_file(build_ctx_path, cid)
-        self._s.build(recipe=recipefile, image=cid, build_folder=build_dest, force=True)
+        self._s.build(
+            recipe=recipefile,
+            image=cid,
+            build_folder=build_dest,
+            force=True,
+            quiet=self._s.quiet,
```
Comment (ivotron) [inline src/popper/runner_host.py:493]: cool! it's nice that there's already a `quiet` parameter on the singularity python lib side. The only thing to note here is that the value for this parameter should be the one from `self._config.quiet`, which is the one that holds the value provided from the CLI

**Review by ivotron — CHANGES_REQUESTED** [review]

**Comment by ishanashah** [comment]
Thanks for the advice! I fixed the logging level changes, sorry about that. I also changed the parameter value to be associated with the command line argument.

**Comment by ivotron** [comment]
thanks @ishanashah!

**Pull request merged**